make docs fails on module import

relates to #12.

### env

- latest OSX
- Python 3.5.1 (same error in Python 2.7)

### log

```
callisto-core [master] » make docs
rm -f docs/callisto-core.rst
rm -f docs/modules.rst
sphinx-apidoc -o docs/ callisto
Creating file docs/callisto.rst.
Creating file docs/callisto.delivery.rst.
Creating file docs/callisto.delivery.management.rst.
Creating file docs/callisto.delivery.management.commands.rst.
Creating file docs/callisto.delivery.migrations.rst.
Creating file docs/callisto.evaluation.rst.
Creating file docs/callisto.evaluation.management.rst.
Creating file docs/callisto.evaluation.management.commands.rst.
Creating file docs/callisto.evaluation.migrations.rst.
Creating file docs/modules.rst.
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C docs clean
rm -rf _build/*
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C docs html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.4.5
making output directory...

Exception occurred:
  File "conf.py", line 17, in <module>
    import delivery
ImportError: No module named 'delivery'
The full traceback has been saved in /var/folders/fq/_rst8fw93jl0dqjsc3vhqzrh0000gn/T/sphinx-err-1dlqy8id.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make[1]: *** [html] Error 1
make: *** [docs] Error 2
```

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -14,7 +14,7 @@
 import os
 import sys
 
-import delivery
+from callisto import delivery
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the

